core/core-spacemacs.el

```diff
@@ -202,8 +201,6 @@ the final step of executing code in `emacs-startup-hook'.")
     (unless (frame-parameter nil 'fullscreen)
       (toggle-frame-maximized))
     (add-to-list 'default-frame-alist '(fullscreen . maximized)))
-  (spacemacs|unless-dumping
-    (dotspacemacs|call-func dotspacemacs/user-init "Calling dotfile user init..."))
```

Comment:
`spacemacs|unless-dumping` calls should be replaced with their bodies.  This change accidentally causes `dotspacemacs/user-init` to never get called.

Reply:
You're correct, the statement is restore in my latest change, thanks!